pyteal/ast/txn.py

```diff
@@ -108,6 +108,7 @@ class TxnField(Enum):
     num_logs = (59, "NumLogs", TealType.uint64, False, 5)
     created_asset_id = (60, "CreatedAssetID", TealType.uint64, False, 5)
     created_application_id = (61, "CreatedApplicationID", TealType.uint64, False, 5)
+    last_log = (62, "LastLog", TealType.bytes, False, 6)
```

Comment:
You could add `StateProofPK` as well, it's the last item in the table at: https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/TEAL_opcodes.md#txn-f

Reply:
@jasonpaulos Thanks for the documentation pointer.  I added `StateProofPK` via https://github.com/algorand/pyteal/pull/174/commits/a181b737aefd63edcf821837a9b6845c991e64c2.

However, I'm mostly unfamiliar with state proof feature details.  So, I welcome feedback/pointers on additional, relevant docstring remarks.

Reply:
If I got a vote, I think it is correct. What we are doing (PyTeal) is not quite related with stateproof, essentially we are building a correspondence to Teal code, so exposing the `StateProofPK` field (and adding corresponding test) seems right to me.